apps/web/src/scenes/CommunityPage/CommunityPageViewHeader.tsx

```diff
@@ -185,7 +212,12 @@ export default function CommunityPageViewHeader({ communityRef, queryRef }: Prop
     const buttonText = showExpandedDescription ? 'Show Less' : 'Show More';
 
     return (
-      <StyledDescriptionWrapper gap={8}>
+      <StyledDescriptionWrapper
+        gap={8}
+        onClick={handleEditClick}
```

Comment:
I noticed the editor opens if I click on "Show More" or "Show Less" on the description. could we block that event so it doesn't trigger the editor if I'm toggling the description expansion? should be the same for clicking on links in the description too.

![CleanShot 2024-02-02 at 21 10 17](https://github.com/gallery-so/gallery/assets/80802871/4e452b25-b5e6-46a0-9229-b2c96385edf6)